packages/typography/src/to-theme.js

```diff
@@ -47,7 +47,24 @@ export const getSpace = (result, opts) => {
   return [0, 1 / 4, 1 / 2, 1, 2, 4, 8].map(v => v * n)
 }
 
-const stackFonts = fonts => fonts.map(font => `"${font}"`).join(', ')
+// genericFontFamilies, wrapFontFamily adapted from typography.js
+// Wrap font names in quotes, unless the font name is actually a keyword.
+// See https://stackoverflow.com/a/13752149 and https://www.w3.org/TR/CSS2/fonts.html#font-family-prop
+const genericFontFamilies = [
+  'inherit',
+  'default',
+  'serif',
+  'sans-serif',
+  'monospace',
+  'fantasy',
+  'cursive',
+  '-apple-system',
```

Comment:
Adding this since it's meant to be the more standard name
```suggestion
  '-apple-system',
  'system-ui',
```